add more changelog entries (#25)
plentymarkets/plugin-plentyshop-lts-modern: meta/documents/changelog_en.md

- It's no longer possible to submit a plentyShop contact form without accepting the privacy policy.
- On mobile, display errors no longer inhibit the usability of the permanently displayed search.
- On mobile, you can now close the language detection from the **Language detection** widget again.

> - On mobile, the images in the **Article grid** widget are displayed correctly again with the setting: "Number of articles per row (mobile): 2".

Reviewer: ```suggestion
- On mobile, the images in the **Article grid** widget are displayed correctly again with the setting: **Number of articles per row (mobile): 2**.
```
**suggestion (non-blocking): Formatting**

All UI elements should be in bold. Doesn't matter if it's the name of a widget, a label or a setting.

Reviewer: ```suggestion
- On mobile, the **Article grid** widget displays images correctly again with the setting: **Number of articles per row (mobile): 2**.
```
**nitpick: Active voice**

Prefer active voice in English.

> - On mobile, it is now ensured that the items in the shopping cart are always visible.

Reviewer: ```suggestion
- On mobile, items in the shopping cart are now always visible.
```
**nitpick: Active voice**

Prefer active voice in English.
